api/docs/vscode-azureresources-api.d.ts

```diff
@@ -491,5 +492,4 @@ export declare interface Wrapper {
     unwrap<T>(): T;
 }
 
-export { };
-
+export { }

```

Comment:
Nit: is this line necessary at all, since this file isn't generated?

Reply:
This file is generated and it just changes all the time super annoying